scriptworker/test/test_cot_verify.py

```diff
@@ -1109,9 +1109,17 @@ def test_check_num_tasks(chain, num, raises):
 # verify_docker_worker_task {{{1
 @pytest.mark.asyncio
 async def test_verify_docker_worker_task(mocker):
-    mocker.patch.object(cotverify, 'check_interactive_docker_worker', new=noop_sync)
-    mocker.patch.object(cotverify, 'verify_docker_image_sha', new=noop_sync)
-    await cotverify.verify_docker_worker_task(mock.MagicMock(), mock.MagicMock())
+    m = mock.MagicMock()
+    m2 = mock.MagicMock()
+    check = mock.MagicMock()
+    mocker.patch.object(cotverify, 'check_interactive_docker_worker', new=check.method1)
+    mocker.patch.object(cotverify, 'verify_docker_image_sha', new=check.method2)
+    await cotverify.verify_docker_worker_task(m, m)
```

Comment:
Nit: not that in matters from a functionality perspective, but I'd rename `s/m/chain` and `s/m2/link`. And at this statement we could call it `cotverify.verify_docker_worker_task(chain, link=chain)`. But I could be wrong.

Reply:
Good call on the renames; done. I think it might not break things to use link=chain, but it's an arg not a kwarg, so it's probably less correct. I'm calling it with `cotverify.verify_docker_worker_task(chain, chain)` .

Reply:
Yeah, makes more sense that way 👍